tests/test_graphical_units/test_img_and_svg.py

```diff
@@ -7,26 +7,241 @@
 from ..utils.testing_utils import get_scenes_to_test
 from ..utils.GraphicalUnitTester import GraphicalUnitTester
 
-# Tests for the modules image_mobject and svg_mobject
+from ..helpers.graphical_units import set_test_scene
 
+# To update the reference object after a visual inspection, use this line:
+# set_test_scene(NameOfTestScene, "img_and_svg")
```

Comment:
```suggestion
```

Reply:
I'd like to include this line so future editors don't have to dive deep into the code to accept the changes of a new test. In my (admittedly limited) experience, image-output tests can be fragile.

Reply:
I believe this line could find a home in the `template_generate_graphical_units_data.py` file, but not really in this file.

Reply:
fixed 219dbd1